wire/controlmsgs_test.go

```diff
@@ -17,6 +17,7 @@ package wire_test
 import (
 	"testing"
 
+	_ "perun.network/go-perun/wire/perunio/serializer" // wire serialzer init
```

Comment:
Could we move this file to wire/perunio/test since it seems to be specific to perunio?

Reply:
Yes, it is specific to `perunio`. But, the tests are in wire package because the Encode/Decode functions are implemented in this package.